RobHouse.java

```diff
@@ -4,12 +4,37 @@
 // Any problem you faced while coding this : no
 
 
+//================================= Approach-1 ================================================
```

Comment:
time and space complexity?

Reply:
I just added the time and space complexity for approach-1 as well

Reply:
Great